website/blog/2023-10-31-to-defer-or-to-clone.md

```diff
@@ -0,0 +1,118 @@
+| **Will data in the target schema drift from data in the source schema?** | No, since deferral will always point to the latest version of the source schema                                                    | Yes, since clone is a point-in-time operation                                          |
+| **Can I use multiple source schemas at once?**                           | Yes, defer can dynamically switch between source schemas e.g. ref unchanged models from production and changed models from staging | No, clone copies objects from one source schema to one target schema                   |
+
+## Should I defer or should I clone?
+
+Putting together all the points above, here’s a handy cheat sheet for when to defer and when to clone:
+
+|                                                                           | defer | clone |
+|---------------------------------------------------------------------------|-------|-------|
+| **Save time & cost by avoiding re-computation**                           | ✅     | ✅     |
+| **Create database objects to be available in downstream tools (e.g. BI)** | ❌     | ✅     |
+| **Safely modify objects in the target schema**                            | ❌     | ✅     |
+| **Avoid creating new database objects**                                   | ✅     | ❌     |
+| **Avoid data drift**                                                      | ✅     | ❌     |
+| **Support multiple dynamic sources**                                      | ✅     | ❌     |
+
+To absolutely drive this point home:
+
+1. If you send someone this cheatsheet by linking to this page, you are deferring to this page
+2. If you print out this page and write notes in the margins, you have cloned this page
+
+## Putting it in practice
+
+Using the cheat sheet above, let’s explore a few common scenarios and explore whether we should use defer or clone for each:
+
+1. **Testing staging datasets in BI**
+
+    In this scenario, we want to:
+    1. Make a copy of our production dataset available in our downstream BI tool
+    2. To safely iterate on this copy without breaking production datasets
+    
+    Therefore, we should use **clone** in this scenario
+    
+2. **[Slim CI](https://discourse.getdbt.com/t/how-we-sped-up-our-ci-runs-by-10x-using-slim-ci/2603)**
+
+    In this scenario, we want to:
+    1. Refer to production models wherever possible to speed up continuous integration (CI) runs
+    2. Only run and test models in the CI staging environment that have changed from the production environment
+    3. Reference models from different environments – prod for unchanged models, and staging for modified models
+    
+    Therefore, we should use **defer** in this scenario
```

Comment:
might be good to callout that clone can be helpful for handing incremental models in CI though!

Reply:
https://github.com/dbt-labs/docs.getdbt.com/issues/4359